s3backgrounddelete/s3backgrounddelete/config/s3_background_delete_config.yaml.sample

```diff
@@ -74,3 +74,4 @@ indexid:
 leakconfig:                                  # Section for object leak config
    leak_processing_delay_in_mins: 15                          # Time delay in mins after which probable delete entry is procesed for leak.
    version_processing_delay_in_mins: 5                        # Time delay in mins after which object version is processed
+   cleanup_enabled: False                                     # Flag for cleanup during S3 instance non-active.
```

Comment:
cleanup_enabled not clearly indicating purpose can this be skip_if_metadata_and_oid_exists = true

Reply:
skip_delete_if_metadata_and_oid_exists can be used but it would be too detailed hence I used cleanup_enabled because its cleanup for oid as well as other cleanups like version_id just to make it simple. Should I make it skip_delete_if_metadata_and_oid_exists?

Reply:
it will be better or at least add more details in comment, that if object metadata and data exists then skip deleting them even if instance is non-active